pkg/controller/installation/products/cloudresources/reconciler.go

```diff
@@ -86,10 +89,46 @@ func (r *Reconciler) Reconcile(ctx context.Context, inst *v1alpha1.Installation,
 		return phase, err
 	}
 
+	phase, err = r.reconcileS3BlobStorage(ctx, inst, client)
+	if err != nil || phase != v1alpha1.PhaseCompleted {
+		return phase, err
+	}
+
 	product.Host = r.Config.GetHost()
 	product.Version = r.Config.GetProductVersion()
 	product.OperatorVersion = r.Config.GetOperatorVersion()
 
 	r.logger.Infof("%s has reconciled successfully", r.Config.GetProductName())
 	return v1alpha1.PhaseCompleted, nil
 }
+
+func (r *Reconciler) reconcileS3BlobStorage(ctx context.Context, installation *v1alpha1.Installation, serverClient pkgclient.Client) (v1alpha1.StatusPhase, error) {
```

Comment:
Which product is this blob storage for, just out of curiosity?

Reply:
it's for the integreatly operator, are the products now responsible for reconciling s3 credentials and bucket? if so I am not sure if we still need to have an blob storage only for the integreatly operator

Reply:
Yeah so the individual products would be responsible for creating the custom resources for the databases/caches that they need. The cloud resource operator picks up on the custom resource and provisions the db and the connection secret that the product will use to connect to the resource

Reply:
Great, so I don't think we need s3 credentials and bucket for the integreatly operator as each individual product creates their own resources. 

ping @philbrookes